.github/workflows/build.yml

```diff
@@ -45,10 +45,12 @@ jobs:
           cache-to: type=gha,mode=max
   build-mac:
     needs: lint
-    runs-on: macos-latest-large
+    # Apparently colima was removed from macos-13-large, so hardcode 12 instead of latest
+    runs-on: macos-12-large
     steps:
       - uses: actions/checkout@v2
-      - uses: docker/setup-buildx-action@v1
+      - uses: douglascamata/setup-docker-macos-action@v1-alpha
```

Comment:
He seems to be using moving target tags for this. We might want to at least lock to `v1-alpha.10`, or to a SHA hash digest instead.

Reply:
Done